httpsign_test.go

```diff
@@ -55,23 +55,27 @@ func TestSignToProxy(t *testing.T) {
 	value := randomString(25)
 	key := []byte(randomString(100))
 	hs := New(key)
-	signature := calcHMAC(key, value)
 	epoch := time.Now().Unix()
+	signature := calcHMAC(key, value, epoch)
 	expectedHeader := formHeader(signature, epoch)
 
 	h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// handle time race condidition if we are right on the second boundary
```

Comment:
Heh, it's funny how you can look back and recognize the signs that you are headed for a mistake...